### ISSUE TITLE
Polygon could not be deleted

### ISSUE THREAD

**Issue by VyacheslavLukin** [innosoft-pro/label-them#33]
During a review, I couldn't delete polygon using Backspace, Delete or any other keys. Is it possible to delete polygon during assignment review?

**Comment by AlNedorezov** [comment]
@VyacheslavLukin, no, it is not possible to delete polygon during assignment review, as Yandex.Toloka doesn't update saved results during the assignment review in general.
To avoid any confusion: during the assignment review reviewer (or Researcher) can ONLY either accept the markup provided by the Mechanical Turk, or decline it.
We are currently working on the release in which all of the editing functionality will be unavailable during the assignment review. We will include this information in the docs during the preparation for this release.

**Comment by VyacheslavLukin** [comment]
Thanks for the information. I suppose we can close this issue when the new version will be released.
